Traceback (most recent call last):
  File "main.py", line 70, in <module>
    main(DEBUG=DEBUG)
  File "main.py", line 58, in main
    cmds.Who(cmd, DEBUG)
  File "/home/pi/ANDY/src/cmds.py", line 272, in __init__
    self.database = Database()
NameError: global name 'Database' is not defined
